utils/src/main/kotlin/Utils.kt

```diff
@@ -198,15 +198,12 @@ fun getPathFromEnvironment(executable: String): File? {
  */
 fun fixupUserHomeProperty() {
     val userHome = System.getProperty("user.home")
-    val checkedUserHome = sequenceOf(
-        userHome,
-        Os.env["HOME"],
-        Os.env["USERPROFILE"]
-    ).find {
-        !it.isNullOrBlank() && it != "?"
+    if (userHome == null || userHome == "?") {
```

Comment:
This omits the check for `userHome` being blank, which I'm basically fine with, but it should probably be documented in the commit message. Or simply continue using `userHome.isNullOrBlank()` here.

Reply:
I have added the blank check again, it matches the functions docs which say "is set to a sane value".